### DISCUSSION TITLE
Latency is dashboard is not populating on Grafana

### DISCUSSION THREAD

**Discussion by guptadeepali39** [Kong/kong#8282] [Help]
I have enabled the Prometheus plugin and using Grafana. All dashboards are appearing but the latency dashboard is coming as blank.

Can someone please guide.

![image](https://user-images.githubusercontent.com/63843149/149044214-383830cd-3314-4b46-a138-38b6356ebc52.png)

**Comment by fffonion** [comment]
Hi @guptadeepali39 is your Kong taking traffic in the timeframe of your graph?
